recipe-generator-node/download-generator-jar.js

```diff
@@ -0,0 +1,25 @@
+const fs = require("fs");
+const https = require("https");
+
+const { FILEPATH, FILE_URL, OUTPUT_DIR, VERSION } = require("./vars");
+
+if (!fs.existsSync(OUTPUT_DIR)) {
+    fs.mkdirSync(OUTPUT_DIR);
+}
+
+console.log(`downloading recipe generator jar ${VERSION} from ${FILE_URL}...`);
+
+const file = fs.createWriteStream(FILEPATH);
+
+https.get(FILE_URL, response => {
+    response.pipe(file);
+    file.on("finish", () => {
+        console.log("finished downloading");
+        file.close();
+    });
+}).on("error", err => {
+    console.log("download failed");
+    console.log(err);
+    fs.unlink();
```

Comment:
What does this do? I'd think it would be `fs.unlink(file)`?

Reply:
Oops yeah I should have the file path in there.

Reply:
Unlink deletes a file.